lib/restrictions.js

```diff
@@ -31,9 +31,9 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
 /**
  * Inserts a restriction.
  *
- * @param {object} options - Options to use.
- * @param {object} options.restriction - An application-specific string that
- *   unambiguously identifies a particular entity to the application.
+ * @param {*} options - Options to use.
+ * @param {object} options.restriction - An application-specific
+ *  string that unambiguously identifies a particular entity to the application.
```

Comment:
Hey I'm seeing this out of context, but just wondering:

> @param {object} options.restriction - An application-specific
>  *  string that unambiguously identifies a particular entity to the application.

shouldn't this be:

```js
@param {string} options.restriction - An application-specific string that unambiguously identifies a particular entity to the application.
```
 
or am I misunderstanding something?

Reply:
@aljones15 good catch. I can update that, thanks. let me know if you know how to write docs for three layer nested object param.

Reply:
This suggestion still isn't quite correct. Looks like it was a copy-paste error from some kind of party identifier or something. A `restriction` is clearly not "An application-specific object that unambiguously identifies a particular entity to the application". We can look at the rest of the docs to see what a restriction is. It looks to be an object that expresses a set of parameters for restricting the acquisition of resources.